src/amo/components/Categories/index.js

```diff
@@ -146,12 +148,21 @@ export class CategoriesBase extends React.Component<InternalProps> {
               // $FLOW_IGNORE
               const { name, slug } = category;
 
+              const linkTo = {
+                pathname: '/search/',
+                query: convertFiltersToQueryParams({
```

Comment:
The results for this new URL seem quite different than the old URL. Is that expected? I was looking at these:

* https://addons.mozilla.org/en-US/firefox/extensions/alerts-updates/
* https://addons.mozilla.org/en-US/firefox/search/?category=alerts-updates&sort=recommended%2Cusers&type=extension

Reply:
Yes, that is expected. We are changing the logic of what gets loaded to use the new sort criteria, so I would expect the results to be different.